lambda/tests/test_instrumentation.py

```diff
@@ -158,6 +158,7 @@ def setUp(self):
             {
                 AWS_LAMBDA_EXEC_WRAPPER: "mock_aws_lambda_exec_wrapper",
                 _HANDLER: "mocks.lambda_function.handler",
+                "AWS_LAMBDA_FUNCTION_NAME": "mylambda",
```

Comment:
This test update is needed after this change upstream: https://github.com/xrmx/opentelemetry-python-contrib/blob/20e8f85985c4aa44cb22585bef1796eb002d55b3/instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py#L435-L437